package.json

```diff
@@ -21,6 +21,6 @@
 		"bs-platform": "1.9.3"
 	},
 	"scripts": {
-		"prepublishOnly": "./bspacks/downloadSomeDependencies.sh && ./bspacks/reason_bspack.sh"
+		"prepublishOnly": "esy dune build js/refmt.bc.js && ./js/closurefy.sh"
```

Comment:
This should be built with release profile

Reply:
So that would be:
```
esy dune build -p reason js/refmt.bc.js
```
correct?
Why does it need release profile?

Reply:
If you’re planning to distribute this for consumption then use release profile. There are many optimizations only enabled on release profile.
 You’re the expert on jsoo by the way.

Reply:
jsoo fails with a stack overflow error in release mode 🤷